### PR TITLE
Customize: add banners page

### PR DESCRIPTION
**Pull request by jennur**
Closes https://github.com/inveniosoftware/docs-invenio-rdm/issues/525

- Adds documentation for how to add new banners in the admin-panel.
- Adds a new section "Features walk-through" in the docs. Lmk if this should be renamed!

### CODE REVIEW

File: docs/features-walk-through/banners.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+**As an admin user** you can add new banners through the admin panel at `/administration`. This is also available through the user-menu in the top right corner of your instance:
+
+<img src="/features-walk-through/img/user-menu-admin.png" width="200"/>
+
+In the admin panel you will see "Banners" as a menu item in the sidebar. If you enter this, you will be shown an overview of all the banners added in your instance. In our case this list is empty, as we didn't add any banners yet.
+
+![Banners page](./img/empty_banners_page.png)
+
+To add a new banner, you can click the green "Create"-button in the top right corner. This will take you to the form where you can add all the details for your new banner:
```
Comment (anikachurilova) [inline docs/features-walk-through/banners.md:15]: "Create" button

File: docs/features-walk-through/banners.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+![New banner form](./img/new_banner_form.png)
+
+Here you can see that the first field "Start time" is already prefilled, however, you are free to update this to whatever start date and time you want.
+
+- "Start time" is when your banner will become visible on your site.
```
Comment (anikachurilova) [inline docs/features-walk-through/banners.md:21]: For Start and End datetimes It's useful to specify that user has to use the UTC format. Can be crucial for some timezones.

File: docs/features-walk-through/banners.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- "End time" is when your banner will deactivate automatically.
+- "Message" is the content that will show in your banner.
+- "Url path" lets you specify where your banner should be visible. If left empty, the banner will show on every page of your site. If you specify a subpath, e.g. `/records`, the banner will show only on `/records` **and all its subpages**.
```
Comment (anikachurilova) [inline docs/features-walk-through/banners.md:24]: Minor: the banner will show only on `/records` **and all its subpages**, e.g. `/records/bla`.

File: docs/features-walk-through/banners.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- "Category" is where you can define the theme of your banner.
```
Comment (anikachurilova) [inline docs/features-walk-through/banners.md:25]: Minor: useful to specify the color-category relation (warning-orange, info-blue ...)

File: docs/features-walk-through/banners.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+- "Active" needs to be checked to activate the banner. This ensures that the banner will show on the given start time and deactivate on the given end time. If left unchecked, the start and end times will be ignored, and the banner will not be visible. This can be used to deactivate the banner temporarily, or permanently.
+
+In the following example, the banner will show up on all subpages of `/records` as an info-banner between April 3rd, 2023 at 12:00 and May 3rd, 2023 at 12:00.
+
+![Filled banner form](./img/filled_banner_form.png)
+
+When ready, click save and you will be taken back to the "Banners" page where the new banner will appear in the list:
```
Comment (anikachurilova) [inline docs/features-walk-through/banners.md:32]: Speak about the search and filtering options for the banners list?

File: docs/features-walk-through/banners.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+![Banner in list](./img/banner_added_list.png)
+
+Here you can see the details of your banner, and whether it is active or not. You can also edit the banner by clicking the "Edit" button in the "Actions" column, or delete it by clicking the trash can.
```
Comment (anikachurilova) [inline docs/features-walk-through/banners.md:36]: Minor: the details of the banner can also be seen on the details page when you click on any banner id in the banners list.

File: docs/features-walk-through/banners.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+If navigating to any page under `/records`, between the given dates, the banner will show above the header on the page:
```
Comment (anikachurilova) [inline docs/features-walk-through/banners.md:38]: It is also possible to add multiple banners on a page. They will appear one above the other.

**Review by anikachurilova — COMMENTED** [review]
Nice work, thank you! Left some comments on additional functional that can be described.

File: docs/features-walk-through/banners.md (hunk 7 above)
Comment (jennur) [reply docs/features-walk-through/banners.md:38]: Wasn't aware! Thanks! 😄

File: docs/features-walk-through/banners.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
+# Site banners
+
+Banners can easily be added to your InvenioRDM instance through the admin panel. They can be added globally or to a specific path.
+
+## Adding a new banner
+
+**As an admin user** you can add new banners through the admin panel at `/administration`. This is also available through the user-menu in the top right corner of your instance:
+
+<img src="/features-walk-through/img/user_menu_admin.png" width="200"/>
```
Comment (kpsherva) [inline docs/features-walk-through/banners.md:9]:
```suggestion
![admin menu](./img/user_menu_admin.png)
```

**Pull request merged**